tests/queries/0_stateless/02377_optimize_sorting_by_input_stream_properties_explain.reference

```diff
@@ -28,145 +22,77 @@ LimitsCheckingTransform × 3
 PartialSortingTransform × 3
 -- ExpressionStep preserves sort mode
 -- QUERY: set optimize_read_in_order=1;set max_threads=3;set query_plan_remove_redundant_sorting=0;EXPLAIN PLAN actions=1, header=1, sorting=1 SELECT a FROM optimize_sorting ORDER BY a
-Sorting (Global): a ASC
 Sorting (Sorting for ORDER BY)
-Sorting (Global): a ASC
-Sorting (Stream): a ASC
-Sorting (Stream): a ASC
+Sorting: a ASC
+Sorting: a ASC
 -- QUERY (analyzer): set optimize_read_in_order=1;set max_threads=3;set query_plan_remove_redundant_sorting=0;EXPLAIN PLAN actions=1, header=1, sorting=1 SELECT a FROM optimize_sorting ORDER BY a
-Sorting (Global): a ASC
 Sorting (Sorting for ORDER BY)
-Sorting (Global): __table1.a ASC
-Sorting (Stream): __table1.a ASC
-Sorting (Stream): a ASC
+Sorting: __table1.a ASC
+Sorting: a ASC
 -- QUERY: set optimize_read_in_order=1;set max_threads=3;set query_plan_remove_redundant_sorting=0;EXPLAIN PLAN actions=1, header=1, sorting=1 SELECT a FROM optimize_sorting ORDER BY a+1
-Sorting (None)
 Sorting (Sorting for ORDER BY)
-Sorting (Global): plus(a, 1) ASC
-Sorting (Chunk): a ASC
```

Comment:
Imho, `Chunk` sorting does not give any helpful information, so I did remove it:
* First of all, I can't guarantee that reading from MergeTree always keeps chunks sorted by PK. Even if so, it is very fragile, e.g. somebody may want to read from different parts in one read task (afiak this is already happening)
* So far, `Chunk` may be used for Sorting (or, maybe, `Distinct` optimization, but I think that's wrong), but in both cases we do in-order optimization and request `Stream` sorting.